ClassLibrary/ClassLibrary/Library.cs

```diff
@@ -41,23 +45,51 @@ public List<Author> Authors
         } 
             = new List<Author>();
 
-        public Author AddAuthor(String nameAuthor,String lastNameAuthor)
+        public void AddAuthorInList(Author author)
```

Comment:
`InList` is not necessary here, because calling code should not care if author is added to a list or anywhere else - it just should stay in the library.